### PR TITLE
fix(orchestrator): pass businessKey for assessments

### PR DESCRIPTION
**Pull request by mareklibra**
backport of https://github.com/janus-idp/backstage-plugins/pull/2558

### CODE REVIEW

**Review by batzionb — APPROVED** [review]

**Comment by batzionb** [comment]
@mareklibra 
to fix prettier run:
cd workspaces/orchestrator
yarn prettier --write .
then commit the changed files

**Comment by mareklibra** [comment]
The prettier is passing when executed in the plugin's folder but not on the workspace level.
We have discrepancy in the dependencies for `prettier`.

I will try to fix that in a separate PR.

**Comment by batzionb** [comment]
meanwhile can you fix it so we can get this merged?
I need to backport https://github.com/janus-idp/backstage-plugins/pull/2573 and it would be difficult without this PR merged

File: workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx [hunk 1]
```diff
@@ -89,7 +90,7 @@ export const ExecuteWorkflowPage = () => {
         const response = await orchestratorApi.executeWorkflow({
           workflowId,
           parameters,
-          businessKey: assessmentInstanceId,
+          businessKey: effectiveInstanceId,
```
Comment (batzionb) [inline workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx:93]:
```suggestion
          businessKey: assessmentInstanceId,
```

File: workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx [hunk 2]
```diff
@@ -98,7 +99,7 @@ export const ExecuteWorkflowPage = () => {
         setIsExecuting(false);
       }
     },
-    [orchestratorApi, workflowId, navigate, instanceLink, assessmentInstanceId],
+    [orchestratorApi, workflowId, navigate, instanceLink, effectiveInstanceId],
```
Comment (batzionb) [inline workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx:102]:
```suggestion
    [orchestratorApi, workflowId, navigate, instanceLink, assessmentInstanceId],
```

File: workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx (hunk 1 above)
Comment (mareklibra) [reply workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx:93]: done

File: workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx (hunk 2 above)
Comment (mareklibra) [reply workspaces/orchestrator/plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx:102]: done

**Review by batzionb — APPROVED** [review]

**Pull request merged**